src/Dropbox/Provider.php

```diff
@@ -13,6 +13,13 @@ class Provider extends AbstractProvider
      */
     public const IDENTIFIER = 'DROPBOX';
 
+    /**
+     * The separating character for the requested scopes.
+     *
+     * @var string
+     */
+    protected $scopeSeparator = ' ';
+
     /**
```

Comment:
Instead of adding it to the docs, please add it as a default scope 👍 

```
protected $scopes = [
   'account_info.read',
];
```

Reply:
You got it.